### ISSUE TITLE
Add species common name

### ISSUE THREAD

**Issue by koonchaya** [rfcx/engineering-support#500]
Original post: https://rfcx.slack.com/archives/C03FD1WD02J/p1705398453628139?thread_ts=1700575002.926709&cid=C03FD1WD02J
From: Tomaz

- Hooded Crow (Corvus cornix) 
- the Edible Frog (Pelophylax esculentus)
- Cinereus Tit (Parus cinereus)

**Comment by grindarius** [comment]
Sorry but follow me up on monday 5th please thank you.

**Comment by rassokhina-e** [comment]
Hi Noon! I checked the comments to this task, I guess we updated the common names manually in the bio DB before, but there is the daily job with IUCN website, so this job rewrited the common names again to the empty values. So, what I did, there is the list of species/common names on the backend side, where we can put the custom data, so I added these 3 species:
const mismatchedNames: Record<string, string> = {
  // Arbimon name : IUCN name
  'Camarhynchus pallidus': 'Geospiza pallida',
  'Camarhynchus heliobates': 'Geospiza heliobates',
  'Corvus cornix': 'Hooded Crow',
  'Pelophylax esculentus': 'the Edible Frog',
  'Parus cinereus': 'Cinereus Tit'
}
To test this case we need to wait the ending of daily sync, this job will finished 40 mins after our daily call tomorrow. (edited)
